deployments/container/Dockerfile

```diff
@@ -14,15 +14,25 @@
 
 ARG GOLANG_VERSION=1.23.8
 # We use an ubuntu20.04 base image to allow for a more efficient multi-arch builds.
-FROM --platform=${BUILDOS}/amd64 nvcr.io/nvidia/cuda:12.8.1-base-ubuntu20.04 AS build
+FROM  nvcr.io/nvidia/cuda:12.8.1-base-ubuntu20.04 AS build
```

Comment:
My assumption when removing `--platform` here was that the default is the current (build) platform.

That assumption was wrong. Ref docs are: https://docs.docker.com/reference/dockerfile/#from

Quote:

> The optional `--platform` flag can be used to specify the platform of the image in case FROM references a multi-platform image. For example, linux/amd64, linux/arm64, or windows/amd64. **By default, the target platform of the build request is used.**